### ISSUE TITLE
.wav or .mp3 don't work

### ISSUE THREAD

**Issue by Radrahil** [cmus/cmus#1228]
Some .mp3 files won't play in cmus (but do in VLC). ```set input.ffmpeg.priority=60``` makes them play properly, however it make .wav files play improperly (sped up and low quality.) Installed from Package Manager (GUI)
---

1) `uname -a`:
```
Linux manjarodb 5.15.78-1-MANJARO #1 SMP PREEMPT Thu Nov 10 20:50:09 UTC 2022 x86_64 GNU/Linux
```

2) `cmus --version`:
```
cmus v2.10.0
Copyright 2004-2006 Timo Hirvonen
Copyright 2008-2016 Various Authors
```

3) `cmus --plugins`:
```
Input Plugins: /usr/lib/cmus/ip
  mad:
    Priority: 55
    File Types: mp3 mp2
    MIME Types: audio/mpeg audio/x-mp3 audio/x-mpeg
  flac:
    Priority: 50
    File Types: flac fla
    MIME Types:
  cue:
    Priority: 50
    File Types:
    MIME Types: application/x-cue
  mpc:
    Priority: 50
    File Types: mpc mpp mp+
    MIME Types: audio/x-musepack
  mp4:
    Priority: 50
    File Types: mp4 m4a m4b
    MIME Types:
  wav:
    Priority: 50
    File Types: wav
    MIME Types:
  opus:
    Priority: 50
    File Types: opus
    MIME Types:
  vorbis:
    Priority: 50
    File Types: ogg oga ogx
    MIME Types: application/ogg audio/x-ogg
  modplug:
    Priority: 50
    File Types: mod s3m xm it 669 amf ams dbm dmf dsm far mdl med mtm okt ptm stm ult umx mt2 psm
    MIME Types:
  wavpack:
    Priority: 50
    File Types: wv
    MIME Types: audio/x-wavpack
  aac:
    Priority: 50
    File Types: aac
    MIME Types: audio/aac audio/aacp
  cdio:
    Priority: 50
    File Types:
    MIME Types: x-content/audio-cdda
  ffmpeg:
    Priority: 30
    File Types: aa aac ac3 aif aifc aiff ape au fla flac m4a m4b mka mkv mp+ mp2 mp3 mp4 mpc mpp ogg opus shn tak tta wav webm wma wv
    MIME Types:

Output Plugins: /usr/lib/cmus/op
  pulse
  alsa
  oss
  jack
  ao

```

**Comment by Charl-pena** [comment]
you didn't find any solution to the problem? I have a similar problem with WavPack files.

**Comment by pgaskin** [comment]
The issue with the `.wav` file with ffmpeg may be fixed by https://github.com/cmus/cmus/pull/1336/commits/406edec3206a228baeaa64e7468387f8c4e8ca1f (part of #1336).

For the `.mp3` file, please provide the output of `ffprobe -i` on it.

**Comment by pgaskin** [comment]
`.wav` is fixed by #1336, re-open this issue if you still have issues with `.mp3.
